src/components/TaskCard/index.jsx

```diff
@@ -1,20 +1,33 @@
 import PropTypes from "prop-types";
+import { useDispatch } from "react-redux";
+import { deleteTask } from "@store/actions";
+import { ICON_DELETE } from "@utils/constants/icons";
+import { ALT_TEXT_DELETE_ICON } from "@utils/constants/texts";
 import { formatDate } from "@utils/helpers/formatDate";
 import "./index.scss"
 
 const TaskCard = ({ task }) => {
-  const { title, createdDate } = task;
+  const { id, title, createdDate } = task;
+  const dispatch = useDispatch();
+
+  const handleDelete = () => {
+    dispatch(deleteTask(id));
+  };
 
   return (
     <div className="task-card">
       <p className="task-card__title">{title}</p>
       <p>Created At: {formatDate(createdDate)}</p>
-    </div>
-  );
+      <button onClick={handleDelete}>
```

Comment:
@hoh-wtag as we are (and will be) repeatedly using this for all CRUD operations, wouldn't it be better if we created a separate component for these buttons having icons?

Reply:
@nsi-wtag bhaia, in this step we are using button with icon just this once. I have created the Icon Button Component in the next step where we are using it multiple times. Let me know if I should do that here as well. Thanks.

Reply:
@hoh-wtag we are using that in both Add Task and Delete Task, if I'm not mistaking. I think it would be better if we could follow that from this step. Thank you!